docs/advanced/security-auditing.md

```diff
@@ -187,6 +187,13 @@ process shutting down.
 
 
 ```yaml title="backend/clutch-config.yaml"
+gateway:
+...
+  middleware:
+    ...
+    // highlight-start
+    - name: clutch.middleware.audit
+    // highlight-end
```

Comment:
Can you also add the config details to https://github.com/lyft/clutch/blob/main/docs/advanced/security-auditing.md#middleware (line 131)

Reply:
Updated! Please take a look when you have a chance.